roles/slurm/files/epilog.d/60-last-user-parts

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 set -ex
 
-num_jobs=$(squeue -h -u "$SLURM_JOB_USER" -w "$HOSTNAME" | wc -l)
+num_jobs=$(squeue -h -u "$SLURM_JOB_USER" -w "$HOSTNAME" -t running | wc -l)
```

Comment:
This command was catching jobs in the `CG` state during the epilog and not running the subsequent task as a result.